tests/fixtures/common.py

```diff
@@ -60,3 +63,15 @@ def create_wells(nrow: int, ncol: int, idomain: xr.DataArray) -> mf6.WellDisStru
     return mf6.WellDisStructured(
         layer=layer, row=iy_active, column=ix_active, rate=rate
     )
+
+
+def create_wells_max_layer(
+    nrow: int, ncol: int, idomain: xr.DataArray
+) -> mf6.WellDisStructured:
+    """
+    Create wells, deactivate inactive cells. This function wouldn't be necessary
+    if iMOD Python had a package to specify wells based on grids.
+    """
+
+    wel_layer = idomain.layer.max().item()
+    return create_wells(nrow, ncol, idomain, wel_layer)
```

Comment:
Maybe something to upstream to iMOD Python? @Huite

Reply:
This docstring is maybe a bit confusing. I copied it from the existing 'create_wells'. I only extended the functionality so it does not always use layer 3 explicitly. I updated the docstring.